geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/index/BinLineStringTest.scala

```diff
@@ -36,14 +36,14 @@ class BinLineStringTest extends Specification with TestWithDataStore {
       val sf = new ScalaSimpleFeature(s"$i", sft)
       val geom = s"LINESTRING(40 6$i, 40.1 6$i, 40.2 6$i, 40.3 6$i)"
       val dates = new java.util.ArrayList[Date]
-      (0 to 4).map(mm => java.util.Date.from(java.time.LocalDateTime.parse(s"2010-05-07T0$i:0$mm:00.000Z", GeoToolsDateFormat).toInstant(java.time.ZoneOffset.UTC))).foreach(dates.add)
+      (0 until 4).map(mm => java.util.Date.from(java.time.LocalDateTime.parse(s"2010-05-07T0$i:0$mm:00.000Z", GeoToolsDateFormat).toInstant(java.time.ZoneOffset.UTC))).foreach(dates.add)
```

Comment:
Ok, so the change from 'to' to 'until' drops out the '4' case.  Does this change in this test show the errors that this PR is trying to fix?  (From reading of the test, I'm trying to see what was happening and why it was wrong;).)